kedro/framework/cli/starters.py

```diff
@@ -545,7 +547,7 @@ def _convert_tool_names_to_numbers(selected_tools: str | None) -> str | None:
     if selected_tools is None:
         return None
     if selected_tools.lower() == "none":
-        return ""
+        return None
```

Comment:
can we combine this with a `or`?

Reply:
Done in [f0ca34f](https://github.com/kedro-org/kedro/pull/3426/commits/f0ca34fbadd8fee939c885ba9f7f19fd12c5891a)